CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 0.8.0
+
+### New Features
+
+- Added support for Twilio Conversations. When an application requests a token from the `/token` endpoint, a `create_conversation=true` parameter can now be specified. When this parameter is present, the `/token` endpoint will create a conversation that is associated with the room and add the participant to it. This allows video application to use Twilio Conversations for in-room chat.
```

Comment:
I think we should add a link to Twilio Conversations here

Reply:
Good idea. Here I'll link to the main page for Twilio Conversations: https://www.twilio.com/conversations-api